src/main/java/com/socialmediaraiser/twitterbot/GoogleAuthorizeUtil.java

```diff
@@ -27,20 +28,15 @@ private GoogleAuthorizeUtil() {
   }
 
   public static GoogleCredential authorize() throws IOException {
-
-    URL yamlFile = GoogleAuthorizeUtil.class.getResource("/RedTheOne.yaml");
-    if (yamlFile == null) {
-      yamlFile = GoogleAuthorizeUtil.class.getResource("/RedTheOne.yaml");
-    }
-    if (yamlFile == null) {
-      LOGGER.severe(() -> "yaml file not found at /RedTheOne.yaml");
-      return null;
+    // @todo use a local json instead
+    URL googleCredentialsFile = GoogleAuthorizeUtil.class.getClassLoader().getResource("google-credentials.json");
+    if (googleCredentialsFile == null) {
+      LOGGER.severe(() -> "file not found");
     }
-    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
Don't instantiate `ObjectMapper` each time.
Use the singleton and call the `readerFor` and `writerFor` methods against it.

Reply:
Yes it was a mistake (check the next line haha)